x-pack/plugins/observability/public/hooks/slo/use_clone_slo.ts

```diff
@@ -80,7 +80,12 @@ export function useCloneSlo() {
         );
       },
       onSuccess: () => {
-        queryClient.invalidateQueries(['fetchSloList']);
+        if (
+          queryClient.getQueryCache().find(['fetchSloList'], { exact: false })?.options // @ts-ignore
+            .refetchInterval === undefined
+        ) {
+          queryClient.invalidateQueries(['fetchSloList'], { exact: false });
+        }
```

Comment:
if we are not currently auto refreshing the view, we don't invalidate the query right? Therefore we let the autorefresh takes care of that?

Reply:
Indeed, that was causing the jumpiness. The polling in `useFetchSlo` collided with the `invalidateQuery` here.